index.html

```diff
@@ -112,7 +112,285 @@
+      "@type": "SoftwareApplication",
+      "name": "ActivityWatch",
+      "url": "https://activitywatch.net/",
+      "description": "The best free and open-source automated time tracker: cross-platform, extensible, and privacy-focused. Ideal for freelancers and designed with ADHD users in mind.",
+      "disambiguatingDescription": "This app is designed for personal task and time management. It helps users efficiently organize their tasks and manage their time effectively. With features tailored to enhance productivity and streamline workflow, it ensures that users can stay on top of their schedules and prioritize their responsibilities with ease.",
+      "keywords": ["time management", "task management", "software", "free auto time tracker", "free freelancer time tracking software"],
+      "abstract": "A collection of trackers that monitor and log key activities on your computer, such as detecting if you are AFK or identifying the active window. A system for storing the data collected by these trackers. A flexible data format that meets most logging requirements. An ecosystem of tools that allows users to customize and extend the software to suit their needs.",
+      "alternativeHeadline": "A free automated time tracker designed to monitor your activity and screen time. This tool helps you keep an accurate record of how you spend your time on your computer, providing insights into your productivity and screen usage patterns. It's an essential tool for improving time management and boosting efficiency.",
+      "about": "ActivityWatch is a bundle of software that tracks your computer activity. You are, by default, the sole owner of your data. It also offers an ecosystem of software to work around it, including ways to collect more data and do different kinds of analysis. There are plenty of companies offering services which collect Quantified Self data with goals ranging from increasing personal productivity to understanding the people that managers manage (organizational productivity). However, all known services suffer from a significant disadvantage: the user’s data is in the hands of the service providers, which leads to a problem of trust. Every customer of these companies has their data in hands they are forced to trust if they want to use their service. This is a significant problem, but the true reason that we decided to do something about it was that existing solutions were inadequate. They focused on short-term insight, a goal worthy in itself, but we also want long-term understanding. We created ActivityWatch completely free and open source so anyone can use, improve and extend it.",
+      "applicationCategory": "https://en.wikipedia.org/wiki/Time-tracking_software",
+      "applicationSubCategory": ["DesktopApplication", "MobileApplication", "WebApplication"],
+      "aggregateRating": {
+          "@type": "AggregateRating",
+          "ratingValue": "4.5",
+          "reviewCount": "11300"
```

Comment:
Not really accurate to use number of stars as review count. Could instead use Google Play Store, or ProductHunt (both linked to on https://activitywatch.net/stats/).